amt/charts/dm-manager/templates/configmap.yaml

```diff
@@ -77,12 +78,31 @@ data:
       exit_istio
     fi
 
+    # Check password policy and set AMT password accordingly
+    PASSWORD_POLICY="{{.Values.params.passwordPolicy | default "dynamic" }}"
+    USER_PASSWORD="{{.Values.params.password | default "" }}"
```

Comment:
When the chart is formatted, this entry would be replaced with the actual password provided, meaning it can be read by dumping out the configmap using kubectl. Is there a way to mask this value in the configmap so it is not shown?